zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -877,7 +919,7 @@ pub fn shield_transparent_funds<DbT, ParamsT, InputsT>(
     from_addrs: &[TransparentAddress],
     min_confirmations: u32,
 ) -> Result<
-    TxId,
+    Vec<TxId>,
```

Comment:
Update the documentation for this function to not assume a single transaction is created.